src/Views/FileTreeView.ts

```diff
@@ -354,23 +361,27 @@ export class FileSystemProvider implements vscode.TreeDataProvider<Entry>,
         element.type === vscode.FileType.Directory ? vscode.TreeItemCollapsibleState.Collapsed :
                                                      vscode.TreeItemCollapsibleState.None);
     if (element.type === vscode.FileType.File) {
+      // TODO Change command to proper command, considering file types
       treeItem.command = {
-        command: 'fileExplorer.openFile',
+        command: 'vscode.open',
         title: 'Open File',
         arguments: [element.uri],
       };
-      treeItem.contextValue = 'file';
+
+      const filePath = element.uri.fsPath;
+      if (filePath.endsWith('.tflite') || filePath.endsWith('.pth') || filePath.endsWith('.onnx')) {
```

Comment:
Maybe `.pb` , `.circle`, `keras-model`, `saved_model` should be added?

Reply:
`pb` and `circle` seems OK.
keras model or saved model is saved into directory, AFAIK. So I will add TODO to check if the dir is for such models.
Also, we have to show backend-specific files... later.
E.g., if the extension of the result of backend-target code is, `bar`,
`bar` should be checked and, e.g., `treeItem.contextValue = targetCode` should be called. 😮